src/core/services/gesture/gesture.js

```diff
@@ -302,7 +302,7 @@ angular.module('material.core')
     onCancel: angular.noop,
     options: {},
 
-    dispatchEvent: dispatchEvent,
+    dispatchEvent: typeof jQuery !== 'undefined' && angular.element === jQuery ? jQueryDispatchEvent : nativeDispatchEvent,
```

Comment:
Maybe we would need to check if jQuery object is really jQuery, but it should work even after merge from what I can read.